src/components/atoms/Selector/SelectorList/SelectorList.tsx

```diff
@@ -13,6 +17,7 @@ export const SelectorList = <OptionValue extends BaseOptionValue>({ options, val
           key={String(option.value)}
           option={option}
           isActive={option.value === value}
+          disabled={option.disabled}
```

Comment:
disabledがoptionalなのでoptional chainingが必要ですかね？

```suggestion
          disabled={option?.disabled}
```

Reply:
あ、確かにそうですね、、
すみません🙏

このままで大丈夫です！